include/roboteam_ai/stp/new_plays/FreeKickThem.h

```diff
@@ -0,0 +1,72 @@
+//
+// Created by jordi on 07-05-20.
+//
+
+#ifndef RTT_FREEKICKTHEM_H
+#define RTT_FREEKICKTHEM_H
+
+#include <stp/Play.hpp>
+
+namespace rtt::ai::stp::play {
+
+class FreeKickThem : public Play {
```

Comment:
Class documentation is missing. since and author tags are still missing. It is important to explain in the class documentation what this play does and why this play is needed, etc.

Reply:
I think the naming makes it clear enough what the play is doing